circle.yml

```diff
@@ -1,17 +1,28 @@
-machine:
-  environment:
-    MINICONDA_PATH: $HOME/miniconda
-    CONDA_ENV_NAME: testenv
+version: 2
 
-dependencies:
-  override:
-    - ./continuous_integration/circle/build_doc.sh:
-        timeout: 3600 # seconds
-test:
-  override:
-    - cat ~/log.txt && if grep -q "Traceback (most recent call last):" ~/log.txt; then false; else true; fi
-
-general:
-  artifacts:
-    - "doc/_build/html"
-    - "~/log.txt"
+jobs:
+  build:
+    docker:
+      - image: circleci/python:3.6.2
+    steps:
+      - checkout
+      - run:
+          command: ./continuous_integration/circle/build_doc.sh
+          no_output_timeout: 60m
+          environment:
+            MINICONDA_PATH: ~/miniconda
+            CONDA_ENV_NAME: testenv
+      - run: 
+          command: |
+            cat log.txt && \
+            if grep -q "Traceback (most recent call last):" log.txt; \
```

Comment:
You don't need this anymore with recent sphinx-gallery. Now the exit code of sphinx-build is non-zero if there is a failure in the examples.